components/resume/editors/PersonalDetailsEditor.tsx

```
@@ -0,0 +1,383 @@
+									userInfo.country?.name
+									? `Address: ${userInfo.address || 'N/A'}, ${
+										userInfo.city || 'N/A'
+									}, ${userInfo.postal || 'N/A'}, ${
+										userInfo.country?.name || 'N/A'
+									}`
+									: 'Address: N/A'}
+							</p>
+							<p className="text-sm">Website: {userInfo.website || 'N/A'}</p>
+							<div className="text-sm">
+								Bio: {userInfo.profile_text ? (
+									<span dangerouslySetInnerHTML={{__html: userInfo.profile_text}} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)